pinc/charsuite-basic-latin.inc

```diff
@@ -11,15 +11,21 @@ $charsuite->codepoints = [
     // without the soft hyphen
     'U+00a1-U+00ac',
     'U+00ae-U+00ff',
-    // codepoints that map to Windows-1252
-    'U+0152', // OE ligature
+    // codepoints that map to Windows-1252 + extras
+    'U+0128', // [~I] (not in CP-1252)
```

Comment:
does php support unicode? We could put the actual characters in the comment too if so.

Reply:
I'm not sure if this answers your question or not, but from the PHP 8.2 manual on strings -- 
>A string is a series of characters, where a character is the same as a byte. This means that PHP only supports a 256-character set, and hence does not offer native Unicode support.

Reply:
I meant in the .php file, not in a string the php language. I wonder if that is more os/git/php interpreter specific. Anyways, it is a minor idea, not even a comment :).

Reply:
Like all things PHP, this is complicated. By default, PHP source files are assumed to be ISO-8859-1 unless [zend.script-encoding](https://www.php.net/manual/en/ini.core.php#ini.zend.script-encoding) is configured on the server or unless [declare(encoding=)](https://www.php.net/manual/en/control-structures.declare.php#control-structures.declare.encoding) is set in the file. We do neither, so the PHP interpreter assumes that all of our scripts are ISO-8859-1 and we can't use anything outside of that for function names, variables, or other "code" structures.

_However_, as Sharon found, PHP just treats strings as a series of bytes, so we could use a literal UTF-8 string inside some PHP string inside a source file and it would happily just treat it as a set of bytes and leave it up to the code as to what to do with it (output it, process it, etc).

During our conversion to UTF-8 we updated all of the dproofreader source files to be explicitly UTF-8 and even have a check as part of our linting that ensures files stay that way (see https://github.com/DistributedProofreaders/dproofreaders/commit/505f00d3fcae558cea6a7f29265a7eec51937254). So we _can_ use UTF-8 characters in source files for strings and comments safely if we want.

Reply:
> does php support unicode? We could put the actual characters in the comment too if so.

I'm perfectly happy to add another commit to update those characters -- it would likely make it easier to read the comments.

Reply:
Characters added to comments and comments harmonized.